mirp/mainFunctions.py

```diff
@@ -122,7 +122,16 @@ def extract_images_for_deep_learning(data_config, settings_config, output_slices
     # Process images for deep learning
     image_list = []
     for data_obj in data_obj_list:
-        image_list += [data_obj.process_deep_learning(output_slices=output_slices)]
+
+        if skip_errors:
+            try:
+                image_list += [data_obj.process_deep_learning(output_slices=output_slices)]
+            except Exception as e:
+                logging.error(f"Failed to extract data for {data_obj.subject}: {e}")
+                pass
```

Comment:
not sure if 'pass' is correct here, I would have expected 'continue', but maybe they have the same effect?

Reply:
Thank you for mentioning this. In fact, `pass` may not even be necessary, as we already print information using `logging.error`. `continue` should not be necessary as the loop concludes after this anyway.

Reply:
You are right :)